libmaxminddb.yaml

```diff
@@ -0,0 +1,48 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libmaxminddb/APKBUILD
+package:
+    name: libmaxminddb
+    version: 1.7.1
+    epoch: 0
+    description: Maxmind GeoIP2 database library
+    target-architecture:
+        - all
+    copyright:
+        - paths:
+            - '*'
+          attestation: TODO
+          license: Apache-2.0
+environment:
+    contents:
+        packages:
+            - busybox
+            - ca-certificates-bundle
+            - build-base
+            - automake
+            - autoconf
+            - curl
+pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: e8414f0dedcecbc1f6c31cb65cd81650952ab0677a4d8c49cab603b3b8fb083e
+        uri: https://github.com/maxmind/libmaxminddb/releases/download/${{package.version}}/libmaxminddb-${{package.version}}.tar.gz
+    - uses: autoconf/configure
+    - uses: autoconf/make
+    - uses: autoconf/make-install
+    - uses: strip
+subpackages:
+    - name: libmaxminddb-static
+      pipeline:
+        - uses: split/static
+      description: libmaxminddb static
+    - name: libmaxminddb-dev
+      pipeline:
+        - uses: split/dev
+      dependencies:
+        runtime:
+            - libmaxminddb
+            - curl
+      description: libmaxminddb dev
+    - name: libmaxminddb-doc
+      pipeline:
+        - uses: split/manpages
+      description: libmaxminddb manpages
```

Comment:
I wonder if we also need a libs subpackage like alpine does?
i.e.
```
    - name: libmaxminddb-libs
      pipeline:
        - runs: |
            mkdir -p ${{targets.subpkgdir}}/usr/lib
            mv ${{targets.destdir}}/usr/lib/libmaxminddb.so.* ${{targets.subpkgdir}}/usr/lib/
      description: libmaxminddb libs
```

Reply:
Added!